esp/src/src-react/hooks/metrics.ts

```diff
@@ -303,18 +314,18 @@ export function useWorkunitMetrics(
         });
     }, [workunit, state, count, scopeFilter, nestedFilter]);
 
-    return [data, columns, activities, properties, measures, scopeTypes, status, increment];
+    return { metrics, columns, activities, properties, measures, scopeTypes, status, refresh: increment };
```

Comment:
Bit of a nitpick I guess, but I might've done `metrics: data` here (and line 378) instead of having the state variable and it's setter have mismatched names.